lib/code_corps/emails/forgot_password_email.ex

```diff
@@ -0,0 +1,14 @@
+defmodule CodeCorps.Emails.ForgotPasswordEmail do
+  import Bamboo.Email
+  import Bamboo.PostmarkHelper
+
+  alias CodeCorps.Emails.BaseEmail
+
+  def create(user) do
+    BaseEmail.create
+    |> to(user.email)
+    |> template(template_id(), {})
```

Comment:
The last parameter here is the substitution data model for the template. The template will have to render a reset password link the client will handle. This means we need to generate the link. This is something we don't really know how it will look until we get to client side, so for now, I would go with the placeholder approach.

```Elixir
|> template(template_id(), [link: link(token)])

# ...

defp link(%AuthToken{value: token}) do
  "#{token}" # TODO: we can replace this with the proper string later
end
```